packages/smooth_app/lib/bottom_sheet_views/user_preferences_view.dart

```diff
@@ -160,8 +160,13 @@ class UserPreferencesView extends StatelessWidget {
         (screenWidth - _TYPICAL_PADDING_OR_MARGIN * 5) * _PCT_ICON;
     final double sliderWidth =
         (screenWidth - _TYPICAL_PADDING_OR_MARGIN * 5) * (1 - _PCT_ICON);
+    final String importanceId = productPreferences.getImportance(attribute.id);
```

Comment:
I was a bit lost when reading those lines of code, e.g. the difference between productPreferences.getImportance() vs productPreferences.getPreferenceImportance() is not obivous from the naming alone.

Could we rename:

final String importanceId = productPreferences.getImportance(attribute.id);
-->
final String importanceId = productPreferences.getImportanceIdForAttributeId(attribute.id);

and:

final PreferenceImportance importance =
        productPreferences.getPreferenceImportance(importanceId);
-->
final PreferenceImportance importance =
        productPreferences.getPreferenceImportanceFromImportanceId(importanceId);